src/components/Blocks/BlockOverview.vue

```diff
@@ -17,17 +17,25 @@ export default {
     components: { BlockCard },
     props: {
         blocks: Array
+    },
+    computed: {
+        sortedBlocks() {
+            let copy = this.blocks;
```

Comment:
const tevens een componemt dient dit niet echt te doen hoort beter thuis bij de mutatie van de blocks in de regatta.js waar ook een test voor is zodat je dit kan uitbreiden met een falende test.

(Block nummers heeft niks met volgorde te maken dat is op tijd)